osu.Desktop/Program.cs

```diff
@@ -22,8 +22,9 @@ public static int Main(string[] args)
         {
             // Back up the cwd before DesktopGameHost changes it
             var cwd = Environment.CurrentDirectory;
+            bool useSdl = args.Contains("--sdl");
```

Comment:
There is no need for the variable since there is only one usage

Reply:
It looks cleaner to me.  If you disagree, make a pull request.